.coveragerc

```diff
@@ -0,0 +1,6 @@
+[report]
+omit =
+    */data/*
```

Comment:
Use `data/*`, since we know the directory is at the top level (same for the others).